pyproject.toml

```diff
@@ -2,11 +2,22 @@
 name = "PyCount"
 version = "0.1.0"
 description = "Simple Python project template"
-authors = [""]
+authors = ["Adriana Solis <solisa@allegheny.edu>, Alexis Caldwell <caldwella2@allegheny.edu>, Rachael Harris <harrisr@allegheny.edu>, Ramon Guzman <guzmanr@allegheny.edu>, Ryan Hilty <hiltyr@allegheny.edu>"]
 license = "MIT"
+packages = [
+    { include = "py_counter.py", from="src" },
+    { include = "src/py_counter.py"},
+]
+
+[tool.poetry.scripts]
+PyCount = "src.command_line_interface:cli"
```

Comment:
The script name should be all lowercase.

Reply:
@RyanHiltyAllegheny to fix script character capitilization